ymcatwincities/openy
# [OYPD-110] Migrate demo content while installation

Task: OYPD-110

### How to review

1. Install site via `reinstall.sh` and you should see `blog` and `branch` demo content.
2. Install site via browser and you should see additional installation step `Import demo content`. Select appropriate content to import.

**NOTE**, `landing` and `programs` content are not migrated (there is no migrations).
